www/kukd01/semestralka1/info.html

```diff
@@ -7,18 +7,11 @@
     <title>Kontakty</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="stylesheet" href="css/style2.css">
```

Comment:
chybi metadata, favicon, prosim precti si pozadavky

Reply:
Chci se zeptat na tento požadavek: ,,Chybí title obsahující název webu + název aktuální stránky"
Mám opravdu přidat na každou stránku nadpis Tamadex s.ro. a h2 třeba úvodní stránka ? Domnívám se že je to zbytečné v kombinaci loga a navigačního menu

Reply:
kdyz je to info.html tak "Tamadex s.r.o. | Informace", uvod.html tak "Tamadex s.r.o. | Uvod", neni to zbytecne